### PR TITLE
Portfolio walkthrough

### PR DESCRIPTION
**Pull request by dangelyjeanelle**
Modified portfolio to include random facts about me as described by the walkthrough

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -13,16 +13,16 @@
...
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['Last year was my first ever doing cheerleading', 'I am a native Spanish speaker', 'I once sprained both of my ankles while walking', 'I know how to speak and write some Mandarin'];
```
Comment (KillianLeC) [inline portfolio/src/main/webapp/script.js:20]: Please format such that this line doesn't scroll horizontally, it helps for reviews.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,19 @@
...
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Dangely's Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>Click here to get a random fact about me:</p>
+      <button onclick="addRandomFact()">About Me</button>
+      <div id="fact-container"></div>
```
Comment (KillianLeC) [inline portfolio/src/main/webapp/index.html:15]: Generally, you don't want to use id's unless this div is the only element that will contain this id. For now it should be fine.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (dangelyjeanelle) [reply portfolio/src/main/webapp/script.js]: I believe I added a new commit to this branch that takes care of this. Was it done properly?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+This repo contains Dangely's portfolio and STEP projects
+
```
Comment (tianxiao25) [inline README.md:2]: remove extra empty lines.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -13,16 +13,19 @@
...
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['Last year was my first ever doing cheerleading',
```
Comment (tianxiao25) [inline portfolio/src/main/webapp/script.js:20]: Are they all real?  :)

Comment (dangelyjeanelle) [reply portfolio/src/main/webapp/script.js:20]: Yes!

**Pull request merged**